lib/ecto/adapters/sql.ex

```diff
@@ -1089,7 +1089,7 @@ defmodule Ecto.Adapters.SQL do
       query: query,
       source: source,
       stacktrace: stacktrace,
-      options: Keyword.get(opts, :telemetry_options, [])
+      options: telemetry_options(opts[:telemetry_options] || [], opts[:schema_migration])
```

Comment:
Oh, this should not be set here, this should be set in the places that set `schema_migration`. The main idea is to avoid having sql now about the special schema_migration option. :)

Reply:
Yeah I think I managed to confuse myself. Do you like what I put in this previous commit: https://github.com/elixir-ecto/ecto_sql/pull/397/commits/557dba4bfa6dcd4c3d35f15efb7448d66d49b9da?

Reply:
Yeah, perfect.

Reply:
Done :). Thanks and sorry about the back and forth.